core-it-suite/pom.xml

```diff
@@ -159,6 +161,42 @@ under the License.
       </testResource>
     </testResources>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.its</groupId>
+        <artifactId>core-it-bootstrap</artifactId>
+        <version>2.1-SNAPSHOT</version>
+        <executions>
+          <execution>
+            <id>boostrap</id>
+            <phase>process-test-classes</phase>
```

Comment:
This might be a total matter of taste, but I think logically `initialize` would be the right phase according to: https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html